server.py

```diff
@@ -38,14 +38,36 @@ def index():
 def list_products():
     category        = request.args.get('category')
     discontinued    = str2bool(request.args.get('discontinued'))
-    results     = []
+    min_price       = request.args.get('min-price')
+    max_price       = request.args.get('max-price')
+    price           = request.args.get('price')
+    limit           = request.args.get('limit')
+    results         = []
+    count           = 0
+    cutoff          = 2
```

Comment:
oh! I put 2 just for a test (since we have two products in the test_server)... I wanted to make it 10.